agent/tool-scripts/postprocess/openshift-metrics-postprocess

```diff
@@ -7,8 +7,8 @@ output_file = sys.argv[2]
 in_csv = []
 metrics=[]
 
-with open(input_file, 'r') as f:
-    reader = csv.reader(f)
+with open(input_file, 'r') as file:
+    reader = csv.reader(null_bytes.replace('\0', '') for null_bytes in file)
```

Comment:
openshift-metrics-datalog script will hit the /metrics endpoint for the duration the benchmark and collects the metrics while the postprocess script formats them in to elasticsearch friendly format. We can't postprocess the data during the start, stop phase.